osidb/models.py

```diff
@@ -2733,6 +2747,83 @@ def __str__(self):
         return self.username
 
 
+class FlawDraft(FlawMixin, AlertMixin):
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created_dt = models.DateTimeField(auto_now_add=True)
+    updated_dt = models.DateTimeField(auto_now=True)
+    origin = models.URLField(max_length=2048)
+    content = models.JSONField()
+    flaw = models.OneToOneField(
+        Flaw, on_delete=models.SET_NULL, related_name="+", null=True, blank=True
+    )
+    checked = models.BooleanField(default=False)
+
+    def _validate_acl_identical_to_parent_flaw(self):
+        # this is a "hack" to prevent this ACLMixin validation from running
+        # because it assumes that:
+        # 1) flaw has a parent relationship to this object which is not
+        #   the case here
+        # 2) flaw is not null, which is not the case here
+        #
+        # the proper solution to this problem would be to decouple said
+        # validation from ACLMixin with e.g. a FlawChildACLMixin mixin
+        # which inherits from ACLMixin and applies the aforementioned
+        # validation and any others related to the Flaw parent relationship
+        ...
+
+    def _validate_embargoed_source(self):
```

Comment:
I would rather move this to the base class to avoid duplicates.

Reply:
The problem is that `FlawMixin` does not inherit from `AlertMixin`, because if it did then we have a diamond inheritance problem because `Flaw` inherits from `BugzillaSyncMixin` which itself depends on `AlertMixin`, so it would not know which implementation of alert/save/etc. to use.

Reply:
But I can document this

Reply:
I see. No problem then. There should not be many validations for the drafts.